GCEWindowsAgent/accounts.go

```diff
@@ -1,4 +1,4 @@
-//  Copyright 2017 Google Inc. All Rights Reserved.
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
This should probably be linux_accounts.go since you have linuxAccountsMgr, would nixAccountsMgr be better?